Makefile

```diff
@@ -70,6 +70,7 @@ setup_lint:
 		--job 'shellcheck' --step 'Install ShellCheck' --no-quiet; \
 	fi
 	pip install jinja2
+	pip install -r tools/linter/clang_tidy/requirements.txt
```

Comment:
isn't this not necessary since we install it in the docker image?

https://github.com/pytorch/test-infra/blob/cb23e13ed41b4ae5cdf158e1fa9888bd0d8515f2/ci-docker-base/Dockerfile.cilint-clang-tidy#L35-L36

Reply:
But this is to run it on user system, isn't it?